forge-game/src/main/java/forge/game/card/CardLists.java

```diff
@@ -414,7 +414,7 @@ public static CardCollection getCardsWithLowestCMC(Iterable<Card> cardList) {
     public static int getTotalPower(Iterable<Card> cardList, boolean ignoreNegativePower, boolean crew) {
         int total = 0;
         for (final Card crd : cardList) {
-            if (crew && StaticAbilityCrewValue.hasAnyCrewValue(crd)) {
+            if (crew) {
```

Comment:
isn't that for "can't crew" effects?

Reply:
CantCrew is another static
this check was just slowing things down
if there's a relevant static there they all need to be checked below anyway for which modifier should apply